content/working_with/console/widgets/deployments.md

```diff
@@ -161,5 +161,11 @@ By default, that page displays the following:
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
 * `Enable click to drill down` - This option enables redirecting to the deployment’s drill down page upon clicking on a specific deployments. Default: On
 * `Show execution status label` - Allows showing last execution workflow ID and status near last execution status icon. Default: Off
+* `Show first user journey buttons` - When there are no installed deployments, it presents a simplified view, as shown below.
+![First user journey buttons]( /images/ui/widgets/blueprint-deployments_first_user_journey.png)
+There are 2 buttons in the view :
```

Comment:
There's small formatting issue :-) 

![image](https://user-images.githubusercontent.com/5202105/159430582-ed1d97d3-ea9b-4627-bfeb-dcce474cdce8.png)

Reply:
Nice catch!
On the smaller screen sizes (like laptop) it seems to be displayed properly, that's why I didn't spoted it
I'll be aware of it in the future, to check docs with possibly the biggest screen available 😁 

Thanks for checking and pointing that out 🎉

Reply:
Necessary changes has been provided 🎸